lib/tessel/restore.js

```diff
@@ -220,7 +220,11 @@ exportables.flash = function(tessel, buffers) {
     buffers.partition = exportables.partition(mac1, mac2);
 
     return Promise.resolve().then(() => {
-        log.info('Bulk Erasing Flash Memory...');
+        log.info('Setting WRITE_ENABLED');
```

Comment:
Maybe if there had been a log statement it wouldn't have been missed when refactoring ;)


Kidding aside, thanks for the merge. Looking forward to contributing.